includes/admin/class-wp-job-manager-settings.php

```diff
@@ -812,14 +812,13 @@ class="regular-text"
 	}
 
 	/**
-	 * Radio input field.
+	 * Radio and Checkbox Outputs.
 	 *
-	 * @param array  $option
-	 * @param array  $ignored_attributes
-	 * @param mixed  $value
-	 * @param string $ignored_placeholder
+	 * @param array  $option Option data.
+	 * @param mixed  $value  Current value.
+	 * @param string $type   'radio' or 'checkbox'.
 	 */
-	protected function input_radio( $option, $ignored_attributes, $value, $ignored_placeholder ) {
+	protected function radio_checkbox_outputs( $option, $value, $type ) {
```

Comment:
Should we also add support for the 'std' option? This is going to be the preselected value if present.

Reply:
I'm not totally sure what you mean by this.  Right now if there is no value the `std` option will show up as the default.